### PR TITLE
feat: DLT-1682 add channel-api icon

### PR DESCRIPTION
**Pull request by fede-dp**
# feat: DLT-1682 add channel-api icon

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [x] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [ ] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1682

## :book: Description

add new icon, asked by the design team for the CC history calls screen

image

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

### CODE REVIEW

File: packages/dialtone-icons/src/icons.json
Comment (ninamarina) [inline packages/dialtone-icons/src/icons.json:1]: This file should not be necessary, let me know if you need help removing it from the PR

Comment (fede-dp) [reply packages/dialtone-icons/src/icons.json:1]: deleted! thanks Nina

**Review by iropolo — APPROVED** [review]

**Review by ninamarina — APPROVED** [review]

**Review by braddialpad — APPROVED** [review]
Thanks!

**Review by yorbi-dp — APPROVED** [review]
Thanks!

**Pull request merged**